tox.ini

```diff
@@ -15,8 +16,9 @@ envlist =
 [gh-actions]
 python =
     3.6: py36-min,py36-pypi
-    3.7: py37-min,py37-pypi,isort-check,isort-check-examples, isort-check-tests,flake8
+    3.7: py37-min,py37-pypi,flake8
```

Comment:
Is there a reason to keep running flake8 on 3.7 instead of 3.9?

Reply:
That old version of flake8 failed on Py39 so I dropped back to Py37 and created a separate flake8 PR.